src/errors/payment_error.rs

```diff
@@ -0,0 +1,9 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum PaymentError {
+    #[error("For BTC wallet, use --sats to specify amount")]
```

Comment:
This should be handled at CLI level.

Reply:
7e229af6ad889eb9bc0791aa572444ce6d9ac221